server/views/scss_style.scss

```diff
@@ -0,0 +1,163 @@
+$cell-color: #fdfdfd;
+$background-color: #f3f3f5;
+
+@mixin inputsize() {
+  text-align: center;
+  font-size: 30px;
+}
+
+body{
+  background-color: $background-color;
+}
+.head{
```

Comment:
use inheritance 
(one selector in another) 
it should be formed as a tree

Reply:
done